PyInstaller/building/build_main.py

```diff
@@ -455,19 +455,38 @@ def __init__(
         # Django hook requires this variable to find the script manage.py.
         CONF['main_script'] = self.inputs[0]
 
+        for path in (pathex or []):
```

Comment:
Think it would be worth constructing a list of offending paths (in case they are multiple), and display that list in a single warning (and later an error)?